ChartController.js

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import {StatsService} from 'bedrock-web-stats';
+
+// TODO: https://www.chartjs.org/docs/latest/charts/
+// implement all chart types
+/**
+ * [ChartTypes from Chart.js](https://www.chartjs.org/docs/latest/charts/)
+ * @typedef {Object} ChartType
+ * @property {string} pie - For Pie charts.
+ * @property {string} time - For Static Time Charts.
+ * @property {string} line - For Static Line Charts.
+ * @property {string} realtime - For realtime Charts.
+ */
+const ChartType = {
+  pie: 'pie',
+  time: 'time',
+  line: 'line',
+  realtime: 'realtime'
+};
+
+/**
+ * Units such as gigabytes that the stats needed to be formatted in.
+ * @typedef ChartUnit
+ * @property {number} gb - the number of bytes in a gigabyte.
+ */
+const ChartUnit = {
+  gb: Math.pow(2, 30)
+};
+
+/**
+ * Receives all the data from each update and get it's ready for the other
```

Comment:
Typo, "its" not "it's".

Reply:
Actually I think it should just be it. will correct this in a moment.